### ISSUE TITLE
Changing url of a WMTS layer

### ISSUE THREAD

**Issue by jampukka** [oskariorg/oskari-docs#295]
Changing url of a `wmtslayer` and then refreshing capabilities from the new endpoint changes both url and the parsed capabilities of the maplayer correctly. However changing them does not alter `options.urlTemplate`, which, if set, determines the actual url to use when requesting tiles from the service.

**Comment by jampukka** [comment]
Yes, that's the one I ran into and almost couldn't figure out why the request still went to the old URL (which the error message kindly told me) even though both the url and the capabilities had been succesfully updated.

I guess the fix is already visioned in the code `// TODO: look at GetLayerCapabilitiesHandler and get resource url from capabilities instead` (https://github.com/oskariorg/oskari-server/blob/master/control-base/src/main/java/fi/nls/oskari/control/layer/GetLayerTileHandler.java#L174).

So the fix would be to remove `options.tileUrl` from having any effect anywhere? If so then it would be nice to add a db migration to remove the field from `options` object from all maplayers (or just `wmtslayer`s). This should be rather safe change to make, I doubt anyone is using `options.tileUrl` for proxied layers to override the URLs from Capabilities.

**Comment by ZakarFin** [comment]
True, true 👍
